rules/ast/ast.go

```diff
@@ -376,6 +376,7 @@ func (node *VectorAggregation) labelsToGroupingKey(labels clientmodel.Metric) ui
 	summer := fnv.New64a()
 	for _, label := range node.groupBy {
 		fmt.Fprint(summer, labels[label])
+		fmt.Fprint(summer, []byte{0})
```

Comment:
I'd suggest clientmodel.Separator as the separator, see https://github.com/prometheus/client_golang/blob/master/model/signature.go#L25

Perhaps
```
    summer.Write([]byte(labels[label]))
    summer.Write([]byte({clientmodel.SeparatorByte})
```
is more efficient than calling fmt.Fprint?